indico/htdocs/js/indico/jquery/categorynavigator.js

```diff
@@ -28,8 +28,22 @@
             openInDialog: false,
             // The title for the category navigator dialog
             dialogTitle: $T.gettext("Select a category"),
-            // Restrict action to categories with no subcategories
-            selectLeafOnly: false,
+            // Allow action to specific categories
+            actionOn: {
+                categoriesWithSubcategories: {
+                    disabled: false,
+                    message: $T.gettext("You cannot select a category that contains subcategories")
+                },
+                categoriesWithEvents: {
+                    disabled: false,
+                    message: $T.gettext("You cannot select a category that contains events")
+                },
+                self: {
+                    disabled: false,
+                    message: $T.gettext("You cannot select the same category"),
```

Comment:
We are talking about 2 things:
* Using the word `"select"`
* Not using the word `"same"`

In the first case, as I explained above, I think `"select"` is a quite generic word since you always need to *select* a category to perform whatever the operation will be. If the majority agrees on something else I'll change it, but I haven't seen a strong argument against it yet.
Secondly, in the suggestions of both of you, you don't give a reason why the category is disabled (`"Disabled for this category"`, `"Not possible for this category?
 "` - but why? I put the word `"same"` because you cannot perform the operation on the *same* categories that you chose (for moving them for example). This is the only (current) usecase anyway.